DockerHostVagrantfile

```diff
@@ -9,6 +9,6 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell", inline:
     "ps aux | grep 'sshd:' | awk '{print $2}' | xargs kill"
 
-  config.vm.network :forwarded_port, host: 8080, guest: 8080
+  #config.vm.network :forwarded_port, host: 8080, guest: 8080
```

Comment:
This was breaking my ability to run the Spring Boot jar off my machine.  If people are using docker for the app server and need this port, I can go back to just commenting it out locally.